pkg/authorization/registry/subjectaccessreview/rest.go

```diff
@@ -99,15 +99,14 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 	requestContext := kapi.WithNamespace(kapi.WithUser(ctx, userToCheck), subjectAccessReview.Action.Namespace)
 	attributes := authorizer.ToDefaultAuthorizationAttributes(subjectAccessReview.Action)
 	allowed, reason, err := r.authorizer.Authorize(requestContext, attributes)
-	if err != nil {
```

Comment:
> if there's an internal error, we don't necessarily know that they aren't allowed... we probably shouldn't return a definitive answer unless allowed is true or err is nil

The signature on Authorize says we'll get back the right answer.  I think this mirrors the signature exactly.  You're saying you'd like to have a remote authorization endpoint that uses a different signature?

What would a non-definitive answer look like via the API?  Seems like the authorizer is the source of truth, so its always definitive.  If you wanted a different answer, give different input, right?